src/subscription/alerts/SubscriptionAlerts.jsx

```diff
@@ -15,8 +15,8 @@ export const SubscriptionAlerts = () => (
   <AlertList
     messageCodes={{
       empty_subscription: (<EmptySubscriptionMessage />),
-      'embargo-error': (<EmptySubscriptionMessage />),
-      'basket-changed-error': (<BasketChangedError />),
+      embargo_error: (<EmptySubscriptionMessage />),
```

Comment:
Can we add `<FallbackErrorMessage />` instead until Product decides what we should display to the learner in this case?

Reply:
> LGTM! Curious on the reason to change `confirmation_status` to `status`?

https://2u-internal.atlassian.net/browse/PON-226

Reply:
> Can we add `<FallbackErrorMessage />` instead until Product decides what we should display to the learner in this case?

done